libs/documentation/src/lib/components/sidenavigation/demos/filter-side-navigation/filter-side-navigation.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="desktop:grid-col-4 tablet-lg:grid-col-12 mobile-lg:grid-col-12 margin-bottom-3">
+  <sds-side-toolbar [responsiveButtonText]="'Advanced Search'" [responsiveDialogOptions]="responsiveDialogOptions"
+    [showApply]=true [dialogTitleText]="'Advanced Search'" [backButtonAria]="'Close Advanced Search'">
+    <ng-template #toolbarContent>
+
+      <div class="sds-card" *ngIf="isMobileMode">
+        <div class="sds-card__header sds-card__header--accent-cool grid-row">
+          <button id="cancelButton" class="sds-button sds-button--circular sds-button--white"
+            (click)="onCancelClicked()" aria-label="Cancel Advanced Search">
+            <usa-icon [icon]="'chevron-left'"></usa-icon>
+          </button>
+          <div class="margin-x-auto">
+            <h2 class="sds-card__title margin-top-1 margin-left-neg-2">Advanced Search</h2>
+          </div>
+        </div>
+      </div>
```

Comment:
Is this mobile mode template needed?

Reply:
Gah looks like I merged something which clobbered my removal of that. Yeah I'll get that removed as it is not needed.